### PR TITLE
Translate some remnant partials for localization

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

**Comment by jimangel** [comment]
Fixes: https://github.com/kubernetes/website/issues/12160

Great work @tengqm! I need to test a little more but initially /lgtm

File: layouts/partials/templates/concept.html [hunk 1]
```diff
@@ -1,8 +1,9 @@
 {{ partial "templates/block" (dict "page" .page "block" "overview" "purpose" "states, in one or two sentences, the purpose of this document") }}
 
+
```
Comment (kbhawkey) [inline layouts/partials/templates/concept.html]: @tengqm, I guess this line is not needed?

File: i18n/en.toml [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
+
+[main_feedback]
+other = "Feedback"
+
+[main_feedback_question]
+other = "Was this page helpful?"
+
+[main_feedback_yes]
+other = "Yes"
+
```
Comment (kbhawkey) [inline i18n/en.toml]: @tengqm, looks good. Is there a reason why the variables are prefixed with `main_`?

File: layouts/partials/deprecation-warning.html [hunk 3]
```diff
@@ -3,8 +3,9 @@
   <main>
     <div class="content deprecation-warning">
       <h3>
-        Documentation for Kubernetes {{ .Param "version" }} is no longer actively maintained. The version you are currently viewing is a static snapshot.
-        For up-to-date documentation, see the <a href="{{ .Site.Params.currentUrl }}">latest</a> version.
+	 Kubernetes {{ .Param "version" }}
+	 {{ T "deprecation_warning" }}
+	 <a href="{{ .Site.Params.currentUrl }}">{{ T "main_latest" }}</a>。
       </h3>
```
Comment (kbhawkey) [inline layouts/partials/deprecation-warning.html]: @tengqm, do you need `version` after `latest`? I was unable to preview the deprecation warning, as those versions are not re-generated?

**Comment by bradtopol** [comment]
Assigning @chenopis as he is the most appropriate person to review this

File: layouts/partials/deprecation-warning.html (hunk 3 above)
Comment (kbhawkey) [reply layouts/partials/deprecation-warning.html]: Sure. The original deprecation string ended as `... latest version.` instead of just `latest`.

**Comment by tengqm** [comment]
@chenopis ping.

File: layouts/partials/deprecation-warning.html [hunk 4]
```diff
@@ -3,8 +3,9 @@
   <main>
     <div class="content deprecation-warning">
       <h3>
-        Documentation for Kubernetes {{ .Param "version" }} is no longer actively maintained. The version you are currently viewing is a static snapshot.
-        For up-to-date documentation, see the <a href="{{ .Site.Params.currentUrl }}">latest</a> version.
+	 Kubernetes {{ .Param "version" }}
+	 {{ T "deprecation_warning" }}
+	 <a href="{{ .Site.Params.currentUrl }}">{{ T "latest_version" }}</a>
```
Comment (chenopis) [inline layouts/partials/deprecation-warning.html]: I'll let you decide, but you could leave the period outside of the string variable and put it after the </a> link tag.

Comment (chenopis) [reply layouts/partials/deprecation-warning.html]: Nevermind. I see now that you need to use the Chinese period for the translation.

**Pull request merged**